docs/firmware/golioth-firmware-sdk/firmware-upgrade/fw-update-zephyr.md

```diff
@@ -55,17 +55,17 @@ west build -b nrf9160dk/nrf9160/ns --sysbuild examples/zephyr/fw_update
 west flash
 ```
 
-By default this will build and run version `v1.2.3` firmware on the Nordic
+By default, this will build and run firmware version `1.2.3` on the Nordic
 nRF9160 development kit.
 
-:::
-
 ### 3. Rebuild firmware with new version number
 
-Now update the firmware version in the project `prj.conf` file:
+Now update the firmware version in the project `VERSION` file:
 
 ```cfg
-CONFIG_MCUBOOT_IMGTOOL_SIGN_VERSION="1.2.4"
+VERSION_MAJOR = 1
+VERSION_MINOR = 2
+PATCHLEVEL = 4
 ```
```

Comment:
Perhaps we should have the old way in a [tab](https://docusaurus.io/docs/markdown-features/tabs) here and below for some time for users that aren't on the latest SDK?

Reply:
Thank you @trond-snekvik, makes sense!

Reply:
@trond-snekvik, added tabs and described how to change the firmware version number with SDK 0.15.0+  and with versions earlier than v0.15.0.